doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -9,9 +9,9 @@ Introduction
 
 MATLAB® and NumPy/SciPy have a lot in common. But there are many
 differences. NumPy and SciPy were created to do numerical and scientific
-computing in the most natural way with Python, not to be MATLAB® clones.
+computing in the most natural way with Python, not to be MATLAB clones.
 This page is intended to be a place to collect wisdom about the
-differences, mostly for the purpose of helping proficient MATLAB® users
+differences, mostly for the purpose of helping proficient MATLAB users
 become proficient NumPy and SciPy users.
```

Comment:
Suggested wording:

MATLAB® and NumPy have a lot in common, but NumPy  was created to
work with Python, not to be a MATLAB clone. This guide will help MATLAB 
users get started with NumPy.

(Unable to submit this as a "suggested change"; GitHub says "Applying suggestions on deleted lines is not supported.")

Reply:
I incorporated the change.